### PR TITLE
Created MPU6050_6Axis_MotionApps20.cpp

### PR DESCRIPTION
**Pull request by onliner10**
Moved all the implementation from MPU6050_6Axis_MotionApps20.h to MPU6050_6Axis_MotionApps20.cpp which used to cause linker errors (redefinition of functions)

### CODE REVIEW

File: Arduino/MPU6050/MPU6050_6Axis_MotionApps20.cpp [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+    uint8_t buf[dmpPacketSize];
+    for (uint8_t i = 0; i < numPackets; i++) {
+        // read packet from FIFO
+        getFIFOBytes(buf, dmpPacketSize);
+
+        // process packet
+        if ((status = dmpProcessFIFOPacket(buf)) > 0) return status;
+        
+        // increment external process count variable, if supplied
+        if (processed != 0) (*processed)++;
```
Comment (marvin2k) [inline Arduino/MPU6050/MPU6050_6Axis_MotionApps20.cpp]: you fixed it ,-)

**Comment by ShahafEdri** [comment]
hey, i just uploaded your code in to my program. good job.

i was wondering what was your considurations on making the functions outside a Class (MPU6050 class for example) (for example -->>bool MPU6050::dmpPacketAvailable();)

**Comment by Pluscrafter** [comment]
Can you also do it with the MPU6050_9Axis_MotionApps41.h I had the same problem with that header and moved the implementations to the cpp file.

**Comment by jrowberg** [comment]
Cleaning up ancient pull requests in this repo; this change has been submitted in different PRs a few times now, and it needs to happen. But there have been enough changes to the files that it needs to be re-done, so I'll close this PR for now and make the changes separately.

**Pull request closed without merging**